src/components/Books.jsx

```diff
@@ -0,0 +1,12 @@
+// import React, { useState } from 'react';
+
+const Books= () => {
+
+  return (
+    <div>
+      <h3>All Books</h3>
+    </div>
+  );
+};
+
+export default Books;
```

Comment:
- [ ] It looks like you have the book list, but at this point, you need to add another important component inside it. Please, create a book component and add it to this booklist. 🐱 
- Renders a list of books using the Book component
- [ ] Remember, the requirements for the book component:
* Takes the book data as a prop and renders the title and the author
* Has a button for removing books
  * the button wouldn't remove the books *yet*, this is just a placeholder for now